demonstrations_v2/tutorial_how_to_use_qualtran_with_pennylane/metadata.json

```
@@ -0,0 +1,39 @@
+{
+    "title": "How to use qualtran with pennylane",
+    "authors": [
+        {
+            "username": "austingmhuang"
+        }
+    ],
+    "dateOfPublication": "2025-07-15T00:00:00+00:00",
```

Just adding some new metadata fields. We are still supporting the starts-with `tutorial_` convention for executing demos, but now you can also use the metadata to specify if the demo is executable in master (stable) and dev (latest). This way, if it stops working in one of the branches, you don't need to rename the demo 🥳 
```suggestion
    ],
    "executable_stable": true,
    "executable_latest": true,
    "dateOfPublication": "2025-07-15T00:00:00+00:00",
```